.ocamlformat

```diff
@@ -3,6 +3,8 @@ break-separators=before
 dock-collection-brackets=false
 break-sequences=true
 doc-comments=before
+wrap-comments = true
+parse-docstrings = true
```

Comment:
Should come in a separate PR.

Reply:
Actually, we should just copy the config in the dune repo both here and in lsp. It will give us a consistent style.